lib/tools.js

```diff
@@ -6,6 +6,7 @@ const format = require('quick-format-unescaped')
 const { mapHttpRequest, mapHttpResponse } = require('pino-std-serializers')
 const SonicBoom = require('sonic-boom')
 const onExit = require('on-exit-leak-free')
+const cloneDeep = require('lodash.clonedeep')
```

Comment:
In addition to Matteo's comment, we would not pull _any_ lodash dependencies into this project. In particular, the `lodash.*` packages are no longer maintained (last I knew). You should look at https://www.npmjs.com/package/rfdc